ci/pipelines/deploy-to-test.yml

```diff
@@ -977,6 +985,50 @@ jobs:
         params:
           image: image/image.tar
           additional_tags: tags/tags
+
+  - name: run-frontend-e2e
+    plan:
+      - get: frontend-candidate-ecr-registry-test
```

Comment:
We could put the 2 gets (this and pay-ci) together in_parallel which speeds up test startup. We don't need to though.